tests/ckeditor/ckeditor.js

```diff
@@ -48,6 +48,14 @@ describe( 'create', function() {
 		} );
 	} );
 
+	it( 'should set configurations on the new editor', function() {
+		var CKEDITOR = modules.ckeditor;
+
+		return CKEDITOR.create( content, { test: 1 } ).then( function( editor ) {
+			expect( editor.config.test ).to.equals( 1 );
```

Comment:
> Man you really can't resist using to.equals, huh? My inner grammar nazi cries every time I read this

:D I always found it weird as well, but I thought we agreed on that, that's why I was using it. Certainly my mistake.

Let's come to a final agreement (please open an issue, not here) and change this in all tests after we close this PR.